client/me/purchases/manage-purchase/auto-renew-toggle/auto-renew-disabling-dialog/index.jsx

```diff
@@ -168,9 +168,12 @@ class AutoRenewDisablingDialog extends Component {
 			>
 				<h2 className="auto-renew-disabling-dialog__header">{ translate( 'Before you go…' ) }</h2>
 				<p>{ description }</p>
-				<Button onClick={ this.onClickGeneralConfirm } primary>
+				<Button onClick={ this.onClickGeneralConfirm }>
 					{ translate( 'Confirm cancellation' ) }
 				</Button>
+				<Button onClick={ onClose } primary>
+					{ translate( "I'll keep it" ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'I'll Keep It' )` *ES Score: 11*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/cancel-button-on-toggling-off-dialog)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).